server/aws-lsp-codewhisperer/src/language-server/netTransform/transformHandler.ts

```diff
@@ -0,0 +1,255 @@
+import path = require('path')
+import { Workspace } from '@aws/language-server-runtimes/out/features'
+import * as fs from 'fs'
+import * as os from 'os'
+import { v4 as uuidv4 } from 'uuid'
+import {
+    CreateUploadUrlResponse,
+    GetTransformationRequest,
+    StopTransformationRequest,
+} from '../../client/token/codewhispererbearertokenclient'
+import { CodeWhispererServiceToken } from '../codeWhispererService'
+import { getCWStartTransformRequest, getCWStartTransformResponse } from './converter'
+import {
+    CancellationJobStatus,
+    QNetCancelTransformRequest,
+    QNetCancelTransformResponse,
+    QNetGetTransformPlanRequest,
+    QNetGetTransformPlanResponse,
+    QNetGetTransformRequest,
+    QNetGetTransformResponse,
+    QNetStartTransformRequest,
+    QNetStartTransformResponse,
+} from './models'
+import { cleanup, createZip, getSha256 } from './utils'
+import fetch = require('node-fetch')
```

Comment:
This won't be able to be bundled in the browser. As a rule of thumb, we don't want to pin any server to a specific platform or architecture.

We can use https://github.com/sindresorhus/ky or other cross-platform fetch wrapper to handle this in a cross platform way.